packages/dx-react-grid/docs/reference/local-selection.md

```diff
@@ -0,0 +1,29 @@
+# LocalSelection Plugin Reference
+
+A plugin that performs local data selection.
+
+## User Reference
+
+### Dependencies
+
+- [SelectionState](selection-state.md)
+
+## Plugin Developer Reference
+
+### Imports
+
+Name | Plugin | Type | Description
+-----|--------|------|------------
+selection | Getter | Set&lt;number &#124; string&gt; | Selected rows.
+rows | Getter | Array&lt;any&gt; | Rows to be filtered.
+getRowId | Getter | (row: any) => number &#124; string | A function used to get a unique row identifier.
+isGroupRow | Getter | (row: any) => boolean | A function used to identify a group row within ordinary rows.
```

Comment:
Should not the `LocalGrouping' and 'CustomGrouping' plugins be listed as optional dependencies?